pkg/migrations/messages/20240419172419_remove-unused-indexes-pt-1.up.sql

```diff
@@ -0,0 +1,4 @@
+SET statement_timeout = 0;
+
+DROP INDEX CONCURRENTLY IF EXISTS messageindex;
```

Comment:
I thought the description said this one can't be killed because it's a primary key?